lib/web/ui/integration.go

```diff
@@ -669,3 +669,35 @@ type AWSOIDCCreateAWSAppAccessRequest struct {
 	// Labels added to the app server resource that will be created.
 	Labels map[string]string `json:"labels"`
 }
+
+// AWSRolesAnywhereListProfilesRequest contains the list of Roles Anywhere Profiles.
+type AWSRolesAnywhereListProfilesRequest struct {
+	// NextToken is the token to be used to fetch the next page.
+	// If empty, the first page is fetched.
+	NextToken string `json:"nextToken"`
+}
+
+// AWSRolesAnywhereListProfilesResponse contains the result of the ListProfiles request.
+type AWSRolesAnywhereListProfilesResponse struct {
```

Comment:
How does this differ from the similar struct in the `integrations/awsra` package?

Reply:
> The UI client usually prefers to have json keys using camelCase, while our grpc files use snake_case.

you mean the `profiles` field right?

it's been inconsistent. lately i've been noticing more that we return things as is (i've been doing it to) unless we need to create a custom type. it at least saves us having to update the ui backend type if we want to say return/access more fields

if i remember correctly, the main motivation of creating a web UI type was to protect it from api changes, like field deletion or changes?